app/src/app/components/header/Header.tsx

```diff
@@ -21,11 +23,20 @@ export default function Header() {
                 Packit
               </div>
             </NavLink>
-            {isAuthenticated && <LeftNav className="mx-6" />}
+            {isAuthenticated && (
+              <>
+                <div className="mx-3 flex items-center md:hidden">
+                  <NavMenuMobile />
+                </div>
```

Comment:
I'm not sure you want to exclude the whole NavMenuMobile if not authenticated, because it also includes Accessibility which should always be visible.